arangod/V8Server/v8-actions.cpp

```diff
@@ -389,13 +389,15 @@ static v8::Handle<v8::Object> RequestCppToV8(v8::Isolate* isolate,
   serverArray->ForceSet(AddressKey, TRI_V8_STD_STRING(isolate, info.serverAddress));
   TRI_GET_GLOBAL_STRING(PortKey);
   serverArray->ForceSet(PortKey, v8::Number::New(isolate, info.serverPort));
+  TRI_GET_GLOBAL_STRING(EndpointKey);
+  serverArray->ForceSet(EndpointKey, TRI_V8_STD_STRING(isolate, Endpoint::uriForm(info.endpoint)));
   TRI_GET_GLOBAL_STRING(ServerKey);
   req->ForceSet(ServerKey, serverArray);
 
   TRI_GET_GLOBAL_STRING(PortTypeKey);
   req->ForceSet(
       PortTypeKey, TRI_V8_STD_STRING(isolate, info.portType()),
-      static_cast<v8::PropertyAttribute>(v8::ReadOnly | v8::DontEnum));
+      static_cast<v8::PropertyAttribute>(v8::ReadOnly));
```

Comment:
@jsteemann If this was non-enumerable for a reason, I can undo this change. But making it unenumerable doesn't reliably hide it (it's still visible via `getOwnPropertyNames`) and the arangod request object is an implementation detail as far as Foxx is concerned anyway so all of its properties are considered "internal" in Foxx, even if they are enumerable.

Making it read-only doesn't prevent overwriting it in JS either, it can still be redefined using `Object.defineProperty`.